src/index.html

```diff
@@ -11,6 +12,30 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      method="POST"
+      class="form_for_search"
+    >
+      <input
+        type="text"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="first_search_bar bar"
```

Comment:
```suggestion
        class="large_search_bar bar"
```
It's not clear what `first` or `second` should mean